reference/constraints/When.rst

```diff
@@ -163,14 +163,16 @@ validation of constraints won't be triggered.
 To learn more about the expression language syntax, see
 :doc:`/reference/formats/expression_language`.
 
-Depending on how you use the constraint, you have access to 1 or 2 variables
+Depending on how you use the constraint, you have access to 1 or 3 variables
 in your expression:
 
 ``this``
     The object being validated (e.g. an instance of Discount).
 ``value``
     The value of the property being validated (only available when
     the constraint is applied to a property).
+``context``
+    This is the context that is used in validation
```

Comment:
Could you add a versionadded directive for this new variable ?

Reply:
We need to improve this description. Right now it's basically saying "context is the validation context" 😐 

For example, in the Validation article (https://symfony.com/doc/current/validation.html) the word `context` is not even mentioned once. So, what's the "validation context"? Any article that we could link from here so readers can learn more? Thanks!